activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter/src/test/java/org/activiti/starter/tests/runtime/ProcessInstanceIT.java

```diff
@@ -0,0 +1,262 @@
+/*
```

Comment:
@gmalanga starters are not longer here in this repository, so you will need to apply your changes in activiti-cloud-starters repository for the relevant starters that you have changed. I imagine, that you are going to provide a new activiti-cloud-query-elastic-starter or something along those lines right?